system.c

```diff
@@ -343,14 +345,52 @@ uint8_t system_execute_line(char *line)
 
         break;
       }
-        case 'P': // Set force sensitivity parameter. Avoids hastle of changing eeprom settings.
+        case 'P': // Digital Potentiometer Settings
+        {
           char_counter++;
-          if (!read_float(line, &char_counter, &parameter)){
+          float dev_id;
+
+          /* Read the number which determines which pot we're talking to */
+          if (!read_float(line, &char_counter, &dev_id)){
             return(STATUS_BAD_NUMBER_FORMAT);
           }
-          settings.force_sensor_level = (uint8_t)parameter;
-          adjustForceSensorPWM();
+
+          switch ((int)dev_id) {
+          case AD5121_0:
```

Comment:
This indentation differs from the higher level switch statement's indentation. I prefer this since it is part of the kernel standard, but we might want to keep it consistent.